### PR TITLE
Implementing gitops command

### PR DESCRIPTION
**Pull request by luizbafilho**
**What changed?**
- Moves `install` to `gitops` command
- Moves install code to `services/gitops` service
- Implements uninstall command

**Why?**
- to comply with the new architecture
- changes in the ux
- new feature

**How did you test it?**
Run:
`wego gitops install`
`wego gitops uninstall`

**Release notes**

**Documentation Changes**

### CODE REVIEW

File: pkg/services/gitops/manifests/app-crd.yaml [hunk 1]
```diff
@@ -0,0 +1,38 @@
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
```
Comment (jpellizzari) [inline pkg/services/gitops/manifests/app-crd.yaml:2]: It seems odd that this is now in the `pkg` directory. Shouldn't this be in `config/crd`?

File: pkg/services/gitops/install.go [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+}
+
+func (g *Gitops) Install(params InstallParams) ([]byte, error) {
+	present, err := g.kube.FluxPresent()
+	if err != nil {
+		return []byte{}, errors.Wrap(err, "could not verify flux presence in the cluster")
+	}
+
+	if present {
+		return []byte{}, fmt.Errorf("Weave GitOps does not yet support installation onto a cluster that is using Flux.\nPlease uninstall flux before proceeding:\n  $ flux uninstall")
```
Comment (jpellizzari) [inline pkg/services/gitops/install.go:21]: :+1: nice error message

File: pkg/runner/runner.go [hunk 3]
```diff
@@ -26,6 +33,56 @@ func (*CLIRunner) Run(c string, args ...string) ([]byte, error) {
 	return cmd.CombinedOutput()
 }
 
+// Run will use exec.Command as a backing medium and run CombinedOutput.
+func (*CLIRunner) RunWithOutputStream(c string, args ...string) ([]byte, error) {
```
Comment (jpellizzari) [inline pkg/runner/runner.go:37]: This looks familiar. Was it moved from somewhere?

**Review by jpellizzari — APPROVED** [review]

File: pkg/runner/runner.go (hunk 3 above)
Comment (luizbafilho) [reply pkg/runner/runner.go:37]: yes

File: pkg/services/gitops/manifests/app-crd.yaml (hunk 1 above)
Comment (luizbafilho) [reply pkg/services/gitops/manifests/app-crd.yaml:2]: indeed, I'll update to get your changes.

**Pull request merged**